advanced/ei-pattern-1/auth.json

```diff
@@ -7,4 +7,4 @@
 			"auth": "reg.auth"
 		}
 	}
-}
+}
```

Comment:
@NishikaDeSilva add a new line EOF.

Reply:
@chirangaalwis we faced an issue after adding a newline for auth.json in APIM. Should we keep it that way?